src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -101,7 +103,10 @@ object Task {
       case TaskStateOp.MesosUpdate(_, taskStatus, now) =>
         updatedHealthOrState(status.mesosStatus, taskStatus.mesosStatus) match {
           case Some(newStatus) =>
-            val updatedTask = copy(status = status.copy(mesosStatus = Some(newStatus)))
+            val updatedTask = copy(
+              networking = networking.copy(networkInfos = taskStatus.networkInfos),
```

Comment:
Just to be sure, 
* it is not possible that these networkInfos change without a Mesos State change, correct?
* all networkInfos are sent with every statusUpdate? Not only if these change?